VR Maze/Assets/Scripts/IMazeGenerator.cs

```diff
@@ -10,6 +10,6 @@ namespace Assets.Scripts
     /// </summary>
     interface IMazeGenerator
     {
-        Maze generate(int width, int height);
+       // Maze generate(int width, int height);

```

Comment:
Originally this was causing build issues after I changed the class structure I meant to un-comment because I fixed the problem.  I'll go change this.